src/Leela.cpp

```diff
@@ -305,7 +305,9 @@ void init_global_objects() {
     // improves reproducibility across platforms.
     Random::get_Rng().seedrandom(cfg_rng_seed);
 
-    NNCache::get_NNCache().set_size_from_playouts(cfg_max_playouts);
+    // When visits are limited ensure cache size is still limited.
+    auto playouts = std::min(cfg_max_playouts, cfg_max_visits);
```

Comment:
I believe it's currently legal to set -v 3200 -p 0 and have this mean 3200 visits with no playout limit. At some point AutoGTP and the server made use of this because -p 1600 was defaulted somewhere in AutoGTP and needed to be overruled.

This already went wrong here, but this change would make the problem worse.

Reply:
Thanks for pointing that out, I hadn't spotted it. 
I'll change to how `-p 0` is handled so it is defaulted to int max when the arg is parsed. Then the two places that use `cfg_max_playouts` can be changed to avoid potential overflow as they each need.

Reply:
Fixed elsewhere.